src-backbone/app/js/controllers/authController.js

```diff
@@ -1,9 +1,11 @@
-let App            = require('utils/sanaAppInstance');
-let Helpers        = require('utils/helpers');
-let AuthLayoutView = require('views/auth/authLayoutView');
-let SignupView     = require('views/auth/signupView');
-let LoginView      = require('views/auth/loginView');
-let SettingsView   = require('views/auth/settingsView');
+let App                       = require('utils/sanaAppInstance');
```

Comment:
use `const` for importing (I didn't want to touch every file when I made the convention switch)